release-notes/cabal-install-3.16.0.0.md

```diff
@@ -0,0 +1,170 @@
+### THIS iS A WIP CHANGELOG FOR 3.16
+
+**It will have to be updated with whaever gets added between 3.14 and 3.16**
+
+
+- Clarify error message when pkg-config is not found [#10122](https://github.com/haskell/cabal/pull/10122)
+
+  - The error message when pkg-config is not found or querying it fails will no
+  longer incorrectly claim that the package is missing in the database.
+
+### Significant changes
+
+- `haddock-project` support for subcomponents [#9821](https://github.com/haskell/cabal/pull/9821)
+
+  - `haddock-project` handles sublibraries, test suites and benchmarks.
+  - `haddock` receives `--package-name` flag whcih allows to set names of
+    components which are included in the main `index.html` file.
+  - added `--use-unicode` flag to `haddock` and `haddock-project` commands.
+  - The directory structure of `./dist-newstyle` has changed.  `haddock`
+    subcommand will install `package:sublib` component in a directory
+    `package/sublib` under `l/sublib/doc/html/`.  This is important for
+    `haddock-project` command and in the future might will be useful for hackage
+    support of sublibraries.  See
+    https://github.com/haskell/cabal/pull/9821#discussion_r1548557115.
+
+### Other changes
+
+- Add support for building profiled dynamic way [#4816](https://github.com/haskell/cabal/issues/4816) [#9900](https://github.com/haskell/cabal/pull/9900)
+
+  Add support for profiled dynamic way
```

Comment:
This detail doesn't add anything. Should it be dropped?

Reply:
```suggestion
```